modules/related-posts/jetpack-related-posts.php

```diff
@@ -1331,10 +1317,15 @@ protected function _generate_related_post_image_params( $post_id ) {
 				}
 			}
 
-			if ( !empty( $img_url ) ) {
-				$image_params['width'] = $thumbnail_size['width'];
+			if ( ! empty( $img_url ) ) {
+				if ( isset( $image_params['alt_text'] ) ) {
```

Comment:
Shouldn't we be checking for `$post_image['alt_text']` here?

```suggestion
				if ( ! empty( $post_image['alt_text'] ) ) {
```